src/ASM/ASMs2DInterpolate.C

```diff
@@ -427,11 +436,14 @@ quasiInterpolation(const Go::BsplineBasis& basis_u,
         int starti = b*(2*q-1);
         int startj = a*(2*p-1);
         int endi, endj;
-        if (invyl == 1) {
-          if (a == 0) {
+        if (invyl == 1)
+        {
+          if (a == 0)
+          {
             endi = starti+2*q-1;
             endj = (startj+p)*perknot;
-          } else if (a == invxl-1) {
+          } else if (a == invxl-1)
```

Comment:
Hvis du vil gjøre alt dette, kan du like gjerne sette inn newline mellom "} else" også (here and elsewhere)